### PR TITLE
revert jaxb to stable version

### PR DESCRIPTION
**Pull request by ilaflamme-va**

### CODE REVIEW

File: charon/pom.xml [hunk 1]
```diff
@@ -16,7 +16,7 @@
     <jacoco.coverage>0.67</jacoco.coverage>
     <javax.resource-api.version>1.7.1</javax.resource-api.version>
     <jaxb-api.version>2.4.0-b180830.0359</jaxb-api.version>
-    <jaxb-runtime.version>3.0.2-b01</jaxb-runtime.version>
+    <jaxb-runtime.version>2.4.0-b180830.0438</jaxb-runtime.version>
```
Comment (eclendenning-va) [inline charon/pom.xml:19]: 2.3.1

File: charon/pom.xml [hunk 2]
```diff
@@ -16,7 +16,7 @@
     <jacoco.coverage>0.67</jacoco.coverage>
     <javax.resource-api.version>1.7.1</javax.resource-api.version>
     <jaxb-api.version>2.4.0-b180830.0359</jaxb-api.version>
```
Comment (eclendenning-va) [inline charon/pom.xml:18]: 2.3.1

File: charon/pom.xml (hunk 1 above)
Comment (ilaflamme-va) [reply charon/pom.xml:19]: This combination does not work. I reverted back to what we previously had functioning.

File: charon/pom.xml (hunk 2 above)
Comment (ilaflamme-va) [reply charon/pom.xml:18]: I don't think we need to go down from `2.4.0-...` to `2.3.1`. I'm reverting back to what we used before I borked it up.

**Review by jhulbert-va — COMMENTED** [review]
Should we be reverting, or should we be trying to find out why the new version borked it?

**Comment by ilaflamme-va** [comment]
I'd much rather fix QA and not be broken. Then decide what we should do.

**Review by jhulbert-va — APPROVED** [review]
Let's come back around to this after QA has settled down.

**Pull request merged**